web/forms.py

```
@@ -1225,9 +1225,13 @@ class FeedbackForm(forms.Form):
 
 
 class ChallengeSubmissionForm(forms.ModelForm):
+    is_public = forms.BooleanField(
+        required=False, initial=False, label="Make submission public", widget=forms.CheckboxInput()
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)